Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:7
+
+ENV TZ Africa/Algiers
+
+RUN curl -Lo- https://yarnpkg.com/install.sh | bash
+ENV PATH $HOME/.yarn/bin:$PATH
+RUN mkdir $HOME/.yarn-cache
+VOLUME $HOME/.yarn-cache # TODO: best way?
```

Comment:
It is not much of a deal at this point, yes. But let's keep it open for a moment if anybody would elaborate on the issue :)